invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -148,3 +148,29 @@ def truncate_number(value, max_value):
     if int(value) > max_value:
         number = compact_number(value, max_value=1_000_000)
     return number
+
+
+def namespace_url(field):
+    """Get custom field namespace url."""
+    namespace_array = field.split(":")
+    namespace = namespace_array[0]
+    namespace_value = namespace_array[1]
+    namespaces = current_app.config.get("RDM_NAMESPACES")
+
+    if namespaces[namespace] is None:
+        return None
+
+    return namespaces[namespace] + namespace_value
+
+
+def custom_fields_search(field, field_value):
+    """Get custom field search url."""
+    namespace_array = field.split(":")
+    namespace = namespace_array[0]
+    namespaces = current_app.config.get("RDM_NAMESPACES")
+
+    if namespaces[namespace] is None:
```

Comment:
I think here `namespaces` can be empty string and this will fail

Reply:
```suggestion
    if not namespaces[namespace]:
```

We now try to use `None` rather than `""`, but maybe to keep things safe and backwards compat, let's do the above?

Reply:
```suggestion
    if not namespaces.get(namespace):
```

I would also use `namespaces.get()` also, in case we get any weird data in accidentally.